src/rendering/scripts/discover/crawler/playground.json

```diff
@@ -0,0 +1,115 @@
+{
+  "data": {
+    "connector_type": "web_crawler",
```

Comment:
Would it be possible to also index all other pages that are not typed? The sessions/speakers/vendors/sponsors/news listing pages, the about us page, the book tickets page, the privacy policy page, the code of terms and policy page, the FAQ page, the tech support page, become a sponsor, become a vendor, landing page listing page, landing pages, etc. I would like to see them in the all results tab.

Reply:
I see. They can be part of a new type of entity, adding a new "Other" result tab right? The entity model would have just title and content, right? Since not all of them have the same structure, probably we have to create different extractors based on the path. For Landing pages we can have a specific extractor. Same for privacy and terms. But about us and book tickets pages are probably the hardest ones, because they don't have a common structure to extract their "content". Any suggestions?

BTW, become a vendor, become a sponsor, FAQ and tech support pages are pointing to `/`, so there's nothing to crawl from them.

Reply:
It can be a new "Other" data type. I would like those search results displayed in the "all results" tab discussed separately.

They all have a `<main>` HTML element where the content of the page is added. This is what needs to be indexed for those pages. You can exclude the common "Latest news and media" and "Premium sponsors" sections content from indexing like on other pages. Of course, those pages would not have facet values except maybe a "Content/Page type" facet.

For the pages you cannot access from the footer, here are their links:
- /sponsors/Become-a-sponsor
- /faq
- /support
- /contact-us
- /inquires
- /map
- /media
- /media/assets
- /media/center

Even if there is nothing relevant in their `<main>` HTML element, I would like them to be indexed to be found by title

Reply:
Since the "All results" tab is not possible, would it be possible to:
- Move all the news pages to a new data type and tab named "News".
- Index the other pages I listed above, the listing pages, the home page, etc. in the "Content" data type? Display them in a "Pages" tab.

Reply:
@InakiAbete-Reflektion Is this request possible? When do you think it could be done?